code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -110,14 +110,12 @@ meteor_act
 	for(var/obj/item/gear in protective_gear)
 		if(istype(gear) && (gear.body_parts_covered & FACE) && !(gear.item_flags & ITEM_FLAG_FLEXIBLEMATERIAL))
 			return gear
-	return null
 
 /mob/living/carbon/human/proc/check_shields(var/damage = 0, var/atom/damage_source = null, var/mob/attacker = null, var/def_zone = null, var/attack_text = "the attack")
-	for(var/obj/item/shield in list(l_hand, r_hand, wear_suit))
-		if(!shield) continue
-		. = shield.handle_shield(src, damage, damage_source, attacker, def_zone, attack_text)
-		if(.) return
-	return 0
+	for(var/obj/item/shield in (get_held_items() | list(wear_suit)))
```

Comment:
`get_held_items()` appears to return a lazylist, which is safe to iterate but not if you do list ops on it like here. I believe you do this in multiple places, so worth searching the proc and checking.

Reply:
According to Wrench:
````
json_encode(null | list("foo")) => "[\"foo\"]"
````

Reply:
For the sake of clarity I went through and broke these ops out into assignments.